solutions/platinum/kattis-segmentintersection.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: kattis-segmentintersection
+source: Kattis
+title: Line Segment Intersection
+author: Rameez Parwez
+---
+
+[Unoffical Analysis](https://cp-algorithms.com/geometry/segments-intersection.html)
+
+## Implementation
```

Comment:
add a disclaimer that this is basically just the cp algo code spruced up

Reply:
done.